src/main/java/uk/gov/hmcts/reform/pip/subscription/management/repository/SubscriptionRepository.java

```diff
@@ -31,6 +31,13 @@ public interface SubscriptionRepository extends JpaRepository<Subscription, Long
     List<Subscription> findSubscriptionsBySearchValue(@Param("search_type") String searchType,
                                                       @Param("search_value") String searchValue);
 
+    @Query(value = "SELECT cast(id as text), channel, search_type, user_id, court_name FROM subscription",
+        nativeQuery = true)
+    List<String> getAllSubsDataForMi();
+
+    @Query(value = "SELECT cast(ID as text), search_value, channel, user_id, court_name FROM SUBSCRIPTION "
```

Comment:
The word "SUBSCRIPTION" in the query is in upper case whereas the other one is in lower case. Shall we make them consistent? :)

Reply:
Sorted